Tasks/CopyFilesOverSSHV0/sshhelper.ts

```diff
@@ -97,11 +98,26 @@ export class SshHelper {
      * @returns {Promise<string>}
      */
     async uploadFile(sourceFile: string, dest: string): Promise<string> {
+        if (process.platform === 'win32') {
+            dest = dest.replace(/\\/g, '/');
+        }
+
         tl.debug('Upload ' + sourceFile + ' to ' + dest + ' on remote machine.');
+        
         var defer = Q.defer<string>();
         if (!this.sftpClient) {
             defer.reject(tl.loc('ConnectionNotSetup'));
         }
+
+        const remotePath = path.dirname(dest);
+        try {
+            if (!await this.sftpClient.exists(remotePath)) {
+                await this.sftpClient.mkdir(remotePath, true);
```

Comment:
what does true mean here? is this the recursive create option? if so, please always prefer the `parameterName: parameterValue` syntax.